src/CoreWf/Attributes.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly:InternalsVisibleTo("TestCases.Activities")]
```

Comment:
`VoidResult` is internal because consumers don't need to be aware of it. As a result, this assembly attribute was necessary to expose `VoidResult` to the testing library.

Reply:
Removing as it's no longer needed - VoidResult doesn't exist anymore.

Reply:
Revert?